pom.xml

```diff
@@ -33,6 +33,7 @@
         <!-- Dependency versions -->
         <jetty.version>9.4.6.v20170531</jetty.version>
         <jersey.version>2.26-b06</jersey.version>
+        <log4j.version>2.8.2</log4j.version>
```

Comment:
I think it's  better to update your master branch and rebase your logs-api branch on top of it.